src/test/java/com/google/api/codegen/testdata/java_mock_service_impl_library.baseline

```diff
@@ -23,6 +23,7 @@ import com.google.tagger.v1.AddLabelResponse;
 import com.google.tagger.v1.LabelerGrpc.LabelerImplBase;
 import io.grpc.stub.StreamObserver;
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
I just had a look at the google java formatter readme (https://github.com/google/google-java-format) and I think the formatter can be used as a library in java code. So we could apply the format to the code generated by toolkit. This would let us:
- not have extra import in the baseline
- optionally, we could remove the formatting step from artman. This means artman has fewer non-python dependencies, and our toolkit-generated code doesn't get changed by anything (at least in Java)